rfcs/0003-standard-gh-labels.md

```diff
@@ -0,0 +1,110 @@
+
+This consistent vocabulary will enable us to create reporting tools that can be applied to any project using the system and will provide a consistent view into the state of all CDS products. It should also reduce cognitive burden when onboarding to the project. 
+
+The list of proposed baseline labels can be seen in Appendix A
+
+### Product Specific Labels
+
+Each product will be allowed to apply their own unique set of labels alongside the baseline labels. 
+
+
+```yaml
+- Name: High Priority | Haute priorité
+  Description: 
+  Colour: 
+- Name: Medium Priority  | Priorité moyenne
+  Description:
+  Colour:
+- Name: Low Priority | Faible priorité
+  Description:
+  Colour
+- Name: Bug | Bogue
+  Description:
+  Colour:
+- Name: Documentation
+  Description:
+  Colour:
+- Name: Security | Sécurité
+  Description:
+  Colour:
+- Name: Privacy | Vie privée
+  Description:
+  Colour:
+- Name: Wont fix | Ne résoudra pas
+  Description:
+  Colour:
+```
+
+
+### Label Requirements
+
+Every label is bound by the following requirement:  
+
+- A label must be bilingual in the format `en | fr`
+- A label must not use a colour used by a baseline label
+
+It is recommended that labels make sense for the product/project they are being applied to and that they are not relevant to only a very small number of issues but this isn’t mandatory.
+
+### Governance
+
+Governance surrounding labels will occur in Github repos, as labels are defined as json objects and are enforced through Github actions we can use Github to apply a level of lightweight automated workflows. Github's Code owner functionality will be used to manage approvals from owners.
+
+#### Creation, Modification, and Approval
+Label Meta-data is defined by Github and can be found here https://docs.github.com/en/rest/reference/issues#create-a-label
+
+The governance surrounding the creation, modification and approval of labels will depend on what label group is being acted upon. 
+
+**CDS wide labels** will be owned by the head of product management. 
+
+**Product Specific Labels** will be owned by the product manager, this is done by applying a .codeowner file against the labels.js file.
+Github Pull requests can be used to provide a lightweight and transparent approval process. 
+
+Any employee of CDS can open a PR or Issue updating the labels, it is recommended that these are documented with a rationale for change.
+
+#### Github Actions
+The head of SRE will own the CDS label repo minus the label.js file which is owned by product managers.
+
+Product specific label repos will be owned by the business unit’s SRE member or the teams 
+
+## Drawbacks
+CDS Employees are not used to things being dictated from top down so this has a potential to cause friction with employees who are used to doing what they want.
```

Comment:
I don't think it is about dictating and we don't want to dictate anything. The drawback is some product will use the new convention and some not which at the end won't allow us to reach the maximum benefit of this RFC. 

We will have to sit with the teams not using the labels and understand their concerns. 

What do you think?

Reply:
But I think if you offer OLA compliant labels out of the box and easy integration, this will be attractive enough